package.json

```diff
@@ -60,51 +60,19 @@
 		"vuex-router-sync": "^5.0.0"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.5",
-		"@babel/eslint-parser": "7.14.5",
 		"@babel/plugin-proposal-object-rest-spread": "^7.14.5",
```

Comment:
Already within `@babel/preset-env`

Reply:
https://babeljs.io/docs/en/babel-plugin-proposal-object-rest-spread

Reply:
> 
> 
> The issue is that either everyone move to that or no one. Switching versions back and forth is not acceptable :(
> So if some apps breaks, we're kind of blocked :'(

Yes, I know. But I guess, we will only know if something breaks if everyone gives it a try. I am fine to move back to npm6 as an interim solution in case something really breaks. However, we won't be able to stay with npm6 forever anyway.

Reply:
https://github.com/nextcloud/tasks/pull/1640